remove classpath from _old_ MANIFEST, this is now done using the OSGi MANIFEST
--- a/freeplane/META-INF/OLD_MANIFEST.MF
+++ b/freeplane/META-INF/OLD_MANIFEST.MF
@@ -1,11 +1,2 @@
 Manifest-Version: 1.0
 Main-Class: org.freeplane.main.application.FreeplaneMain
-Class-Path: ../
-  freeplaneeditor.jar
-  freeplaneviewer.jar
-  freeplanemac.jar
-  commons-lang-2.6.jar
-  forms-1.0.5.jar
-  jortho.jar
-  gnu-regexp-1.1.4.jar
-  SimplyHTML.jar
